apps/arm/main.cpp

```diff
@@ -180,7 +187,25 @@ static mbed_error_status_t setPIDTuningMode(CANMsg &msg) {
 
 // Configure PID parameters
 static mbed_error_status_t setPIDParameter(CANMsg &msg) {
-    return MBED_SUCCESS; // TODO
+    // ASSUME PAYLOAD IS 32 BITS LONG
+    int32_t payload(0);
+    msg >> payload;
+    // The actuatorID is made of the 4 LSBs of the payload
+    uint8_t actuatorID = (payload & 15);
```

Comment:
You can use a [bitfield](https://en.cppreference.com/w/cpp/language/bit_field) to make this cleaner

Reply:
Assuming u know the endianness, you could reinterpret cast the can msgs data into a bitfield struct

Reply:
does this mean ill be able to consider groups of bits in the payload differently. For example, can i choose to interpret bits[0:3] as a int and [0:32] as a float?

Reply:
should read thru the link i sent ab bitfields. effectively, its interpreted as the type you defined, limited by the number of bits u set